relay-server/src/processing/trace_metrics/store.rs

```diff
@@ -164,6 +164,20 @@ fn attributes(
         },
     );
 
+    // Add key names matching metric name and type to workaround current co-occuring attributes limitations.
+    result.insert(
+        format!("sentry._meta.cooccuring.name.{}", metric_name),
```

Comment:
Was that decided that we want to use `_meta` for everything, I assumed we'd mainly use that for the pii etc. metadata that is directly connected to fields

Reply:
It's either this or `internal`, should be one of the hidden fields, internal might be better 🤔